app/models/emotion.rb

```diff
@@ -12,4 +12,5 @@ class Emotion < ApplicationRecord
     enum category: [:negative, :neutral, :positive]
     validates :word, presence: true, length: { in: 2..15 }, uniqueness: {case_sensitive: false}
     validates :category, inclusion: { in: Emotion::categories }
+    before_save { self.word.downcase! unless self.word.blank? }
```

Comment:
A more elegant way to do the same thing is `self.word&.downcase!`. To the right of the `&` won't run if to the left of it returns nil.